### PR TITLE
Style forms and create dropdown for states

### PR DESCRIPTION
**Pull request by martensonbj**
Made the forms a little bit easier to deal with - less wide, larger fonts, etc. Also added a dropdown for the state input field. I added a helper file to handle all of the state info and then call it in the form_for in the view to avoid making the controller or view pass along each of the states. 

I might have also made Minnesota the permanent default...mwahahaha. We can change that.

### CODE REVIEW

**Comment by martensonbj** [comment]
I'm choosing to ignore the %w request here because it seems irrational to do that for ever array of 2 elements per state. Rebased up until this point.

**Comment by martensonbj** [comment]
Also added a before action to the users model and fixed the tests that originally had a test-field for the state field.

File: app/helpers/states_helper.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    ["Kansas", "KS"],
+    ["Kentucky", "KY"],
+    ["Louisiana", "LA"],
+    ["Maine", "ME"],
+    ["Maryland", "MD"],
+    ["Massachusetts", "MA"],
+    ["Michigan", "MI"],
+    ["Minnesota", "MN"],
+    ["Mississippi", "MS"],
+    ["Missouri", "MO"],
```
Comment (ToniRib) [inline app/helpers/states_helper.rb]: I agree with your statement below,  Brenna.

**Comment by ToniRib** [comment]
Loooooove the new forms, they look so much better! I'm fixing up the hound complaints then will push up and merge this in.

**Comment by ToniRib** [comment]
Also I definitely agree with ignoring Hound for the %w for the arrays, since some of them have two word elements and it would look weird to keep changing back and forth between the two array styles.

**Pull request merged**